src/handlers.js

```diff
@@ -145,8 +145,8 @@ async function jobHandler(message) {
   let sha = message.payload.details['event.head.sha'] || await context.github.repos.getShaOfCommitRef({
     owner: organization,
     repo: repository,
-    ref: `heads/${message.payload.details['event.base.repo.branch']}`,
-  });
+    ref: `refs/tags/${message.payload.details['event.version']}`,
+  }).sha;
```

Comment:
I assume it's the `.sha` that makes the critical difference in fixing the error, as before the URI included some long JSON blob?  At any rate, this looks good.